scout/parse/case.py

```diff
@@ -9,6 +9,7 @@
 
 from scout.constants import PHENOTYPE_MAP, REV_PHENOTYPE_MAP, REV_SEX_MAP, SEX_MAP
 from scout.exceptions import ConfigError, PedigreeError
```

Comment:
ConfigError doesn't look used any more in this file:
```suggestion
from scout.exceptions import PedigreeError
```

Reply:
Fixed!